pkg/client/pipelines.go

```diff
@@ -132,9 +132,7 @@ func JobSpecFromJobSpecFragment(gateName string, jsFragment *console.JobSpecFrag
 		gateNameAnnotationKey := v1alpha1.GroupVersion.Group + "/gatename"
 		jobSpec.Template.ObjectMeta.Annotations[gateNameAnnotationKey] = gateName
 	}
-	ttlSeconds := int32(7200)
```

Comment:
We actually do want a ttl though, it just needs to be long enough that logs are still usable

Reply:
maybe 24h, WDYT?

Reply:
Yea that seems sane to me

Reply:
done